test/data_table.go

```diff
@@ -0,0 +1,87 @@
+package test
+
+import (
+	"fmt"
+
+	"github.com/DATA-DOG/godog/gherkin"
+	"github.com/sergi/go-diff/diffmatchpatch"
+
+	"github.com/Originate/git-town/test/helpers"
+)
+
+// DataTable compares Gherkin tables with user-generated data.
+type DataTable struct {
+	// cells contains table data organized as rows and columns
+	cells [][]string
+}
+
+// FromGherkin provides a DataTable instance populated with data from the given Gherkin table.
+func FromGherkin(table *gherkin.DataTable) (result DataTable) {
+	for _, tableRow := range table.Rows {
+		mortaRow := make([]string, len(tableRow.Cells))
```

Comment:
Bunch of references to morta in here. Is that because this was translated from mortadella? Thoughts on removing those references as may be confusing for any new developers

Reply:
Good catch. I initially named `DataTable` as `Mortadella` to match the name of a comparable Ruby library, but then decided that the name is too dumb. I'm slowly growing up!